build_info/libnet-dev.control

```diff
@@ -0,0 +1,21 @@
+Package: libnet-dev
```

Comment:
For whatever reason Debian has, this should be libnet9-dev.
(Debian is outdated so they are at libnet1-dev)